integration-cli/docker_api_containers_test.go

```diff
@@ -507,7 +507,6 @@ func (s *DockerAPISuite) TestContainerAPIBadPort(c *testing.T) {
 		PortBindings: container.PortMap{
 			"8080/tcp": []container.PortBinding{
 				{
-					HostIP:   "",
```

Comment:
(related to my previous comment, as I know we've had some cases where things were ambiguous -> "" being `0.0.0.0` or not, etc).